azure-functions/src/send_grid/message_builder.rs

```diff
@@ -811,32 +905,42 @@ impl MessageBuilder {
     }
 
     /// Consumes the builder and returns the SendGrid message.
-    pub fn build(self) -> SendGridMessage {
+    pub fn finish(self) -> SendGridMessage {
         self.0
     }
 
+    // PRIVATE: Appends a `to` email address to the SendGrid message addresses.
```

Comment:
I generally don't document small private methods as hopefully the intention is clear from the code alone.

Reply:
Good note. Removed